### PR TITLE
Add Iden3StateInfo2023 type

### PR DESCRIPTION
**Pull request by ilya-korotya**

### CODE REVIEW

File: core/jsonld/auth.jsonld [hunk 1]
```diff
@@ -22,6 +22,92 @@
           "@type": "xsd:string"
         }
       }
+    },
+    "Iden3StateInfo2023": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/auth.json-ld#Iden3StateInfo2023",
```
Comment (vmidyllic) [inline core/jsonld/auth.jsonld:27]: fix links in this files to ```https://schema.iden3.io/core/jsonld/auth.jsonld```

File: core/jsonld/auth.jsonld [hunk 2]
```diff
@@ -22,6 +22,92 @@
...
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "@vocab": "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/state-info.md#",
```
Comment (vmidyllic) [inline core/jsonld/auth.jsonld:34]: copy vocab from credentials to core folder also and change the url

File: core/jsonld/auth.jsonld (hunk 1 above)
Comment (ilya-korotya) [reply core/jsonld/auth.jsonld]: Done

File: core/jsonld/auth.jsonld (hunk 2 above)
Comment (ilya-korotya) [reply core/jsonld/auth.jsonld]: Done

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**